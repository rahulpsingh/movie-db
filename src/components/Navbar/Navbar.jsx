import React from "react";
import styled from "styled-components";
import logo from "../../images/logo.svg";
import Searchbar from "../Searchbar";

const NavbarWrapper = styled.nav`
  width: 1200px;
  margin: 0 auto;
  display: flex;
  justify-content: space-between;
  border-bottom: 1px solid lightgrey;
  padding: 2vh;
`;

const Logo = styled.img`
  cursor: pointer;
`;

const Navbar = ({ setSearchResults }) => {
  const handleClick = () => {};
  return (
    <NavbarWrapper>
      <Logo src={logo} alt="Timescale" onClick={handleClick} />
      <Searchbar setSearchResults={setSearchResults} />
    </NavbarWrapper>
  );
};

export default Navbar;
