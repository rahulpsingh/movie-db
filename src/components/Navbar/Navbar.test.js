import { render } from "@testing-library/react";
import Navbar from "./Navbar";

describe("Navbar", () => {
  it("should render the Navbar", () => {
    const { container } = render(<Navbar toggle={() => {}} movieInfo={{}} visible />);
    expect(container).toMatchSnapshot();
  });
});
