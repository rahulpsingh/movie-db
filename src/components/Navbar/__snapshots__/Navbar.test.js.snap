// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render a modal 1`] = `
<div>
  <nav
    class="sc-hKFxyN biQiEE"
  >
    <img
      alt="Timescale"
      class="sc-eCApnc dKLneu"
      src="logo.svg"
    />
    <div
      class="sc-bdnxRM kARabk"
    >
      <input
        class="sc-gtsrHT iVZvUU"
        placeholder="Search for a movie"
        type="text"
        value=""
      />
      <img
        alt="search-img"
        class="sc-dlnjwi kObtID search-icon"
        src="search.svg"
      />
    </div>
  </nav>
</div>
`;

exports[`Navbar should render the Navbar 1`] = `
<div>
  <nav
    class="sc-hKFxyN biQiEE"
  >
    <img
      alt="Timescale"
      class="sc-eCApnc dKLneu"
      src="logo.svg"
    />
    <div
      class="sc-bdnxRM kARabk"
    >
      <input
        class="sc-gtsrHT iVZvUU"
        placeholder="Search for a movie"
        type="text"
        value=""
      />
      <img
        alt="search-img"
        class="sc-dlnjwi kObtID search-icon"
        src="search.svg"
      />
    </div>
  </nav>
</div>
`;
